test-app/overlays/dev/storefront-deploy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: storefront
+spec:  
+  template:
+    spec:
+      containers:
+      - name: vc-storefront-web
+        volumeMounts:
+        - mountPath: /opt/virtocommerce/storefront/wwwroot/cms-content
+          name: cms-content-data
+        env:
+          - name: CookieAuthenticationOptions__ExpireTimeSpan
+            value: 12:00:00
+          - name: SnapshotCollectorConfiguration__IsEnabled
+            value: "false"
+          - name: APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
Please comment out APPINSIGHTS_INSTRUMENTATIONKEY variable and its value to prevent AppInsight unauthorized use